author.tf

```diff
@@ -231,6 +231,53 @@ module "author-survey-runner-static" {
   memory_units               = "512"
 }
 
+module "author-survey-launcher" {
+  source                   = "github.com/ONSdigital/eq-ecs-deploy?ref=v4.1"
+  env                      = "${var.env}"
+  aws_account_id           = "${var.aws_account_id}"
+  aws_assume_role_arn      = "${var.aws_assume_role_arn}"
+  vpc_id                   = "${module.author-vpc.vpc_id}"
+  dns_zone_name            = "${var.dns_zone_name}"
+  ecs_cluster_name         = "${module.author-eq-ecs.ecs_cluster_name}"
+  aws_alb_arn              = "${module.author-eq-ecs.aws_external_alb_arn}"
+  aws_alb_listener_arn     = "${module.author-eq-ecs.aws_external_alb_listener_arn}"
+  service_name             = "surveys-launch"
```

Comment:
You might find that this service name will clash with the other launcher service when deployed to pre-prod. Wonder if it's worth naming this one `author-launch` (similarly to how we named author-survey-runner) to differentiate between them?

Reply:
Makes sense, I'll do it in this PR just do it's one less thing to remember.